modules/apps/frontend-js/frontend-js-recharts/liferay-npm-bundler.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	"create-jar": false,
+	"exports": {
+		"recharts": "recharts",
+		"recharts_lib_index": "recharts/lib/index.js"
+	},
+	"imports": {
+		"frontend-js-node-shims": {
+			"events": "^1.0.0"
+		},
+		"frontend-js-react-web": {
+			"react": "^16.0.0",
+			"react-dom": "^16.0.0"
+		}
+	},
+	"output": "build/node/packageRunBuild/resources",
+	"work-dir": "build/node/bundler"
```

Comment:
why not workdir?

Reply:
it feels weird with the hyphen, and might induce to errors... 🤔

Reply:
@jbalsas Because all other compound settings use hyphens (see `create-jar` for example).

We are in time to change it before bundler 3 gets released, but we have keep in mind that the more we change, the more difficult the migration from bundler 2 to 3.

Reply:
OK, read it wrong. I thought you were proposing `workDir` :sweat_smile:

Reply:
Will change that.

Reply:
was it `work-dir` in 2? I thought it was a new setting...

Reply:
It was nothing in bundler 2 :sweat_smile: . See my [comment above](https://github.com/andresfulla/liferay-portal/pull/221#discussion_r390823964). I have created an issue to change it.